slim-playbook.yml

```diff
@@ -0,0 +1,71 @@
+site:
+  title: Couchbase Docs
+  url: https://docs.couchbase.com
+  start_page: home::index.adoc
+  robots: allow
+  keys:
+    google_analytics: GTM-MVPNN2
+    nav_groups: |
+      [
+        { "title": "Server", "startPage": "home::server.adoc", "components": ["server"] },
+        { "title": "Mobile", "startPage": "home::mobile.adoc", "components": ["couchbase-lite", "sync-gateway"] },
+        { "title": "Cloud", "startPage": "cloud::index.adoc", "components": ["cloud"] },
+        { "title": "Cloud-Native", "startPage": "cloud-native-database::index.adoc", "components": ["cloud-native-database"] },
+        { "title": "Autonomous Operator", "components": ["operator"] },
+        { "title": "Service Broker", "components": ["service-broker"] },
+        { "title": "SDKs", "startPage": "home::sdk.adoc", "components": ["*-sdk", "cxx-txns", "elasticsearch-connector", "kafka-connector", "spark-connector"] },
+        { "title": "Tutorials", "startPage": "tutorials::index.adoc", "components": ["tutorials"] }
+      ]
+git:
+  ensure_git_suffix: false
+urls:
+  latest_version_segment_strategy: redirect:to
+  latest_version_segment: current
+content:
+  branches: release/*
+  sources:
+  - url: .
+    branches: HEAD
+    start_path: home
+#  - url: https://github.com/couchbase/couchbase-cli.git
+#    branches: [master]
+#    start_path: docs
+#  - url: ../docs-sdk-python/
+#    branches: [HEAD]
+#  - url: ../docs-sdk-dotnet/
+#    branches: [HEAD]
+#  - url: ../docs-sdk-java/
+#    branches: [HEAD]
+#  - url: ../docs-server/
+#    branches: [HEAD]
+asciidoc:
+  attributes:
+    max-include-depth: 10
+    page-partial: false
+    experimental: ''
+    idprefix: '@'
+    idseparator: '-@'
+    sqlpp: SQL++
+    tabs: tabs
+    toc: ~
+    page-toclevels: 1@
+    xrefstyle: short
+    enterprise: https://www.couchbase.com/products/editions[ENTERPRISE EDITION]
+    community: https://www.couchbase.com/products/editions[COMMUNITY EDITION]
+    kroki-fetch-diagram: true
+    kroki-server-url: ~
```

Comment:
We should perhaps use our internal Kroki server here

Reply:
done